semantics/src/main/kotlin/com/strumenta/kolasu/semantics/symbol/description/SymbolDescription.kt

```diff
@@ -0,0 +1,50 @@
+package com.strumenta.kolasu.semantics.symbol.description
+
+import com.strumenta.kolasu.language.KolasuLanguage
+import com.strumenta.kolasu.model.Named
+import com.strumenta.kolasu.model.Node
+
+data class SymbolDescription(
+    override val name: String,
+    val identifier: String,
+    val types: List<String>,
+    val fields: Map<String, ValueDescription>
+) : Node(), Named
```

Comment:
When working on serializing symbols to LionWeb I realized it was not so much work to do so, so we do not need to make symbols into nodes for serialization purposes. Without that reason, do we still it still makes sense to have symbols as nodes?